### PR TITLE
Bug 1798101: Add test cases for VM filtering in VM list view

### PR DESCRIPTION
**Pull request by rhrazdil**
Added test coverage for VM filtering in VM list view implemented during our CNV UX Meetup.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.list-filtering.scenario.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  });
+
+  afterAll(() => {
+    deleteResource(testVM);
+  });
+
+  it('Displays correct count of Other VMs', async () => {
+    await vm.waitForStatus(VM_STATUS.Importing);
+    await vm.navigateToListView();
+    const importingCount = await filterBoxCount('Importing');
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.list-filtering.scenario.ts]: can this function return number already so we don't have to do the conversion each time we call it?

Comment (rhrazdil) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.list-filtering.scenario.ts]: Yes, updated.

**Pull request merged**